lib/cucumber/core/test/result.rb

```diff
@@ -177,6 +177,24 @@ def ok?(be_strict = false)
           end
         end
 
+        class Ambiguous < Raisable
+          include Result.query_methods :ambiguous
+
+          def describe_to(visitor, *args)
+            visitor.failed(*args)
+            visitor.duration(duration, *args)
+            self
+          end
+
+          def to_s
+            "F"
```

Comment:
I agree, I hit this in Cucumber JS the other day, and I wonder if we should just do exactly the same as them to stay consistent (except offering to `--guess`).

💡  in fact, why don't we just _steal_ their acceptance test...

https://github.com/cucumber/cucumber-js/blob/master/features/ambiguous_step.feature